### ISSUE TITLE
Missing trailing comma error on spec file with JSX code

### ISSUE THREAD

**Issue by rfgamaral** [palantir/tslint#2872]
### Bug Report

- __TSLint version__: 5.4.0
- __TypeScript version__: 2.3.4
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
test("answer button renders correctly", () => {
    expect(shallow(
        <CallButton
            type="answer"
            onClick={undefined} /> // this is line 15
    )).toMatchSnapshot();
});
```

with `tslint.json` configuration:

```json
{
    "extends": [
        "tslint:latest",
        "tslint-react"
    ],
    "rules": {
        "jsx-alignment": false,
        "newline-before-return": true,
        "no-duplicate-super": true,
        "variable-name": [true,
            "check-format",
            "ban-keywords"
        ],
        "typedef": [true,
            "call-signature",
            "member-variable-declaration",
            "parameter",
            "property-declaration"
        ]
    }
}
```

#### Actual behavior

> ERROR: tests/components/dialer/call-button/CallButton.spec.tsx[15, 43]: Missing trailing comma

#### Expected behavior

No error.

**Comment by adidahiya** [comment]
this is the expected behavior for multiline call statements.

**Comment by rfgamaral** [comment]
How come? It's a JSX element, having a comma doesn't make much sense.

**Comment by ajafff** [comment]
It's just a multiline call expression. I rearranged your code to make that clearer
```ts
test("answer button renders correctly", () => {
    const element = <CallButton
            type="answer"
            onClick={undefined} />;
    expect(shallow(
         element, // this is a multiline call expression. according to your tslint.json you want to have trailing commas here
    )).toMatchSnapshot();
});
```

**Comment by rfgamaral** [comment]
I see. Would it make sense to tweak the rule to have an option for ignoring the trailing comma if there's only one expression?
